packages/ladle/lib/cli/vite-plugin/generate/get-meta-json.js

```diff
@@ -3,18 +3,24 @@ import { storyIdToMeta } from "../naming-utils.js";
 /**
  * @param entryData {import('../../../shared/types').EntryData}
  */
-const getMetaJson = (entryData) => {
+export const getMetaJson = (entryData) => {
   /** @type {string[]} */
   let storyIds = [];
   /** @type {{[key: string]: any}} */
   let storyParams = {};
   /** @type {{[key: string]: any}} */
   let storyMeta = {};
+  /** @type {{[key: string]: any}} */
+  let namedExports = {};
 
   Object.keys(entryData).forEach((entry) => {
     entryData[entry].stories.forEach(({ storyId, locStart, locEnd }) => {
```

Comment:
```suggestion
    entryData[entry].stories.forEach(({ storyId, locStart, locEnd, namedExport }) => {
```

you should have access to `namedExport` here